app/src/main/java/com/github/se/wanderpals/ui/screens/suggestion/SuggestionBottomSheet.kt

```diff
@@ -40,7 +42,10 @@ import com.github.se.wanderpals.service.SessionManager
  */
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun SuggestionBottomSheet(viewModel: SuggestionsViewModel) {
+fun SuggestionBottomSheet(
```

Comment:
Documentation for the function  `SuggestionBottomSheet` could be added to improve code readability.